UnitsNet.WindowsRuntimeComponent/GeneratedCode/UnitAbbreviationsCache.g.cs

```diff
@@ -1153,17 +1157,29 @@ public sealed partial class UnitAbbreviationsCache
                 ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.AtmospherePerSecond, new string[]{"atm/s"}),
                 ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.AtmospherePerSecond, new string[]{"атм/с"}),
                 ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopascalPerMinute, new string[]{"kPa/min"}),
-                ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopascalPerMinute, new string[]{"кПа/с"}),
+                ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopascalPerMinute, new string[]{"кПа/мин"}),
                 ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopascalPerSecond, new string[]{"kPa/s"}),
                 ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopascalPerSecond, new string[]{"кПа/с"}),
+                ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopoundForcePerSquareInchPerMinute, new string[]{"kipf/in²/min", "ksi/min"}),
+                ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopoundForcePerSquareInchPerMinute, new string[]{"kipf/in²/мин", "ksi/мин"}),
+                ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopoundForcePerSquareInchPerSecond, new string[]{"kipf/in²/s", "ksi/s"}),
+                ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.KilopoundForcePerSquareInchPerSecond, new string[]{"kipf/in²/с", "ksi/с"}),
                 ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.MegapascalPerMinute, new string[]{"MPa/min"}),
-                ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.MegapascalPerMinute, new string[]{"МПа/с"}),
+                ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.MegapascalPerMinute, new string[]{"МПа/мин"}),
                 ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.MegapascalPerSecond, new string[]{"MPa/s"}),
                 ("ru-RU", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.MegapascalPerSecond, new string[]{"МПа/с"}),
+                ("en-US", typeof(PressureChangeRateUnit), (int)PressureChangeRateUnit.MegapoundForcePerSquareInchPerMinute, new string[]{"Mpsi/min", "Mlb/in²/min"}),
```

Comment:
Found a source verifying `ksi` and `Mpsi` are commonly used terms.
https://en.wikipedia.org/wiki/Pounds_per_square_inch